src/app.js

```diff
@@ -291,6 +291,7 @@ const schemaRoutes = [
   ["/schemas/dataset/v2",              "https://github.com/nextstrain/augur/raw/master/augur/data/schema-export-v2.json"],
   ["/schemas/dataset/root-sequence",   "https://github.com/nextstrain/augur/raw/master/augur/data/schema-export-root-sequence.json"],
   ["/schemas/dataset/tip-frequencies", "https://github.com/nextstrain/augur/raw/master/augur/data/schema-tip-frequencies.json"],
+  ["/schemas/dataset/measurements",    "https://github.com/nextstrain/augur/raw/master/augur/data/schema-measurements.json"],
```

Comment:
This github URL doesn't exist (yet). There's https://github.com/nextstrain/augur/pull/827 but it's not yet ready for merge, and it doesn't name the file `schema-measurements.json` either.

Reply:
Ahhh, I now see your commit comment covers this: "Currently gets redirected to a 404 page, but will soon be added as a JSON schema in augur."